backend/dal/Migrations/20200930210531_DisableRoles.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Pims.Dal.Migrations
+{
+    public partial class DisableRoles : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql("UPDATE dbo.Roles SET IsDisabled = 1 WHERE Name in ('Assistant Deputy Minister', 'Assistant Deputy Minister Assistant', 'Executive Director')");
```

Comment:
Move all scripts to the seed folder to be maintained as separate SQL.  You can see how this is accomplished by the `Initial` migration.  This requires you to use the `SeedMigration` instead.